src/ray/core_worker/store_provider/memory_store/memory_store.h

```diff
@@ -91,12 +92,12 @@ class CoreWorkerMemoryStore {
              absl::flat_hash_map<ObjectID, std::shared_ptr<RayObject>> *results,
              bool *got_exception);
 
-  /// Convenience wrapper around Get() that stores ready objects in a given result set.
-  Status Wait(const absl::flat_hash_set<ObjectID> &object_ids,
-              int num_objects,
-              int64_t timeout_ms,
-              const WorkerContext &ctx,
-              absl::flat_hash_set<ObjectID> *ready);
+  /// Waits for a number of objects to be ready from the list of object_ids given.
```

Comment:
This needs to document the semantics of the return value: a set of objects that are ready immediately in the in-memory store and another set of objects that are ready but are in plasma and need to waited on there